Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -244,50 +244,6 @@
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/kilt.rsi
 
-- type: entity
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitChiefJustice
-  name: chief justice's jumpsuit
-  description: A fancy black jumpsuit with a lace cravat to make it even more fancy. Proper judicial attire.
-  components:
-  - type: Sprite
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cj.rsi
-  - type: Clothing
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cj.rsi
-
-- type: entity
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitChiefJusticeFormal
-  name: chief justice's formal jumpsuit
-  description: A fancy double-breasted suit with golden accoutrements. Sharp and authoritative.
-  components:
-  - type: Sprite
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cjformal.rsi
-  - type: Clothing
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cjformal.rsi
```

Comment:
Leave stuff like clothing item assets in, please. They can just be added as loadout items for Lawyers to take.